dodola/cli.py

```diff
@@ -346,6 +346,12 @@ def train_qdm(
     multiple=True,
     help="'key1=value1' entry to merge into the output Dataset root metadata (attrs)",
 )
+@click.option(
+    "--wet-day-post-correction",
```

Comment:
```suggestion
    "--wetday-post-correction",
```

I think this makes it more consistent with the "wetday" dodola command -- it doesn't use a hyphen.